docker_entrypoint.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+if [ "$DATABASE" = "postgres" ]
+then
+    echo "Waiting for the BD..."
```

Comment:
Should probably be "Waiting for the DB..."

Reply:
Ah, my _spanglish_